### PR TITLE
IMP: ANCOM -> ANCOM-BC tutorial updates

### PR DESCRIPTION
**Pull request by lizgehret**
Replaces ancom with ancombc in PD Mice and Moving Pictures tutorials.

### CODE REVIEW

File: source/tutorials/moving-pictures.rst
Comment (cherman2) [inline source/tutorials/moving-pictures.rst:441]:
```suggestion
ANCOM-BC operates on a FeatureTable[Frequency] QIIME 2 artifact. We can run ANCOM-BC on the subject column to determine what features differ in abundance across gut samples of the two subjects.
```

File: source/tutorials/moving-pictures.rst
Comment (cherman2) [inline source/tutorials/moving-pictures.rst:447]: Could you address why you dont need an intercept?

Comment (cherman2) [reply source/tutorials/moving-pictures.rst:447]: I see its addressed alittle later but it think it would be good to explain that it is alphabetical

File: source/tutorials/moving-pictures.rst
Comment (cherman2) [inline source/tutorials/moving-pictures.rst:489]:
```suggestion
.. We see more differential abundant features in the original compared the collapsed table, which is reasonable since we are collapsing at the genus level and thus losing some resolution. However, collapsing at level 6 may allow us to investigate patterns that aren't present when looking at ASVs
```

File: source/tutorials/pd-mice.rst
Comment (cherman2) [inline source/tutorials/pd-mice.rst:618]: WOW! Literally such a GREAT definition

File: source/tutorials/pd-mice.rst
Comment (cherman2) [inline source/tutorials/pd-mice.rst:632]: It looks like your threshold is based on the q-value and not the log fold change value. Is that correct?

File: source/tutorials/pd-mice.rst
Comment (cherman2) [inline source/tutorials/pd-mice.rst:748]:
```suggestion
.. Bespoke provided slightly better resolution overall, which is reasonable since that was the classifier that we trained using animal stool samples.
```

This is stupid and nit-picky but I think you can train classifiers and not taxonomies.  A classifier is a machine learning model that can be trained, while a taxonomy is a static result

**Review by cherman2 — COMMENTED** [review]
This looks really good to me @lizgehret. You do an incredible job of explaining ancom-bc

File: source/tutorials/moving-pictures.rst
Comment (lizgehret) [reply source/tutorials/moving-pictures.rst:447]: good call, i'll include that after the first function command block!

File: source/tutorials/pd-mice.rst
Comment (lizgehret) [reply source/tutorials/pd-mice.rst:632]: yep, good catch - thank you!

**Comment by gregcaporaso** [comment]
@lizgehret, this looks great! 

I made a few little edits in the most recent commit. 
1) "Why?" -> "Why might you expect this?" in the moving pics docs. Since it's not necessarily the case that there would be fewer differentially abundant genera than ASVs, I just felt like "Why might..." was a better way to ask this question. 
2) `'donor + genotype'` -> `'donor * genotype'` - this aligns with how we run the anova earlier in the tutorial, and makes sense because donor and genotype could interact (apologies if I previously suggested using `+` here). 

Also, in the multiple interfaces moving pictures tutorial, the filtering command is rendering strangely:
Screenshot 2024-01-12 at 11 48 36 AM

If you're good with my edits, this is good to merge once we get that rendering issue sorted out.

**Review by gregcaporaso — APPROVED** [review]
Just see my comment above about the filtering command rendering strangely. I don't need to review this again before merge. Thanks @lizgehret!

**Comment by lizgehret** [comment]
Thanks @gregcaporaso! Oh yeah, that does look weird - I'll render locally and see what's going on there!

**Comment by lizgehret** [comment]
> If you're good with my edits, this is good to merge once we get that rendering issue sorted out.

Okay, so for posterity - this has been fixed, and now looks reasonable when rendered locally:
Screen Shot 2024-01-17 at 1 20 29 PM

**Comment by gregcaporaso** [comment]
Thanks @lizgehret!

**Pull request merged**
